src/components/ChattyLLM/ChattyLLMInputForm.vue

```diff
@@ -693,6 +724,17 @@ export default {
 						if (sessionId === this.active.id) {
 							this.active.sessionAgencyPendingActions = responseData.sessionAgencyPendingActions
 							this.active.agencyAnswered = false
+							// update content of previous message if we receive an audio message from the assistant
+							// or if the last human message had an audio attachment
+							if (this.doesLastHumanMessageHasAudio()
+								|| (responseData.role === Roles.ASSISTANT && responseData.attachments.find(a => a.type === SHAPE_TYPE_NAMES.Audio))
```

Comment:
a similar function `this.doesLastAssistantMessageHaveAudio()` would be good to read but it's fine either way